packages/wifi-signal-strength/default.nix

```diff
@@ -0,0 +1,59 @@
+# Copyright 2022-2023 TII (SSRC) and the Ghaf contributors
+# SPDX-License-Identifier: Apache-2.0
+{
+  stdenvNoCC,
+  pkgs,
+  lib,
+  ...
+}: let
+  wifiSignalStrength =
+    pkgs.writeShellScript
+    "wifi-signal-strength"
+    ''
+      export DBUS_SESSION_BUS_ADDRESS=unix:path=/tmp/ssh_session_dbus.sock
+      export DBUS_SYSTEM_BUS_ADDRESS=unix:path=/tmp/ssh_system_dbus.sock
+      ${pkgs.openssh}/bin/ssh -M -S /tmp/nmcli_socket \
+          -f -N -q ghaf@192.168.100.1 \
```

Comment:
please move hard coded literals like `192.168.100.1` under let declarations and use variable here instead
- like 
`some-literal-variable = ghaf@192.168.100.1` 
...
`... -q ${some-literal-variable} \ `

Reply:
It is updated as a variable but didn't pass as an argument because it works with #427 when it is replaced with `net-vm.ghaf`.